pkg/providers/aws/config.go

```diff
@@ -36,7 +36,7 @@ const (
 	sesSMTPEndpointEUWest1 = "email-smtp.eu-west-1.amazonaws.com"
 )
 
-//get default namespace
+//DefaultConfigMapNamespace get default namespace
```

Comment:
[The "Commentary" section of Effective Go](https://golang.org/doc/effective_go.html#commentary) says:
> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared. 

I believe that's the practice that this lint rule is trying to enforce, so I think this could be improved by re-wording -- the old comment was really describing the action of calling `k8sutil.GetWatchNamespace()`, but the new one should be describing `DefaultConfigMapNamespace`.

```suggestion
// DefaultConfigMapNamespace is the default namespace that ConfigMaps will be created in.
```

I think there are a couple of other comment changes in this PR that similarly don't roll off the tongue very well, and could benefit from a re-wording.